controller/predicates/is_admin_only_analysis.js

```diff
@@ -0,0 +1,13 @@
+const _ = require('lodash');
+
+module.exports = (request, response) => {
+  if (!request.clean.hasOwnProperty('parsed_text')) {
+    return false;
+  }
+
+  // return true only if all non-admin properties of parsed_text are empty
+  return ['number', 'street', 'query', 'category'].every((prop) => {
```

Comment:
No, this is dealing with the translated field names, so `house` in libpostal becomes `query` in the API.